src/Lecture2.hs

```diff
@@ -57,8 +63,9 @@ lazyProduct = error "TODO"
 >>> duplicate "cab"
 "ccaabb"
 -}
+
 duplicate :: [a] -> [a]
-duplicate = error "TODO"
+duplicate = concatMap (replicate 2)
```

Comment:
Great! 👍🏼 
If you want to write this function even more elegantly, you can use the standard function `concatMap` which is a combination of `concat` and `map` as the name suggests 🙂
After that, you could notice, that you would be able to eta-reduce on the last argument :)